src/errors.rs

```diff
@@ -17,11 +16,11 @@ pub enum Error<E> {
     ///
     /// Typically this indicates a faulty implementation of `serde::Serialize` or
     /// `serde::Deserialize`.
-    ClientDeserialize(bincode::serde::DeserializeError),
+    ClientDeserialize(::bincode::Error),
```

Comment:
Let's just remove this variant entirely, since the inner types are the same now.